publish.sbt

```diff
@@ -39,4 +31,4 @@ pomExtra := (
         <name>Bruno Ballekens</name>
         <url>https://github.com/brunoballekens</url>
       </developer>
-    </developers>)
```

Comment:
I think we can/should remove the pomExtra as i would assume this is generated with the `ThisBuild / developers`  ?